javalib/src/main/scala/java/util/function/DoublePredicate.scala

```diff
@@ -22,7 +22,8 @@ trait DoublePredicate { self =>
   def and(other: DoublePredicate): DoublePredicate = {
     new DoublePredicate {
       def test(value: Double): Boolean =
-        self.test(value) && other.test(value) // the order and short-circuit are by-spec
+        self.test(value) && other.test(
+            value) // the order and short-circuit are by-spec
```

Comment:
Ugh seriously :(

Reply:
This particular instance was fixed by increasing the line length limit. But in general, it's possible to fix this kind of issue manually in daily development. The thing about scalafmt is that it may not produce the prettiest results on the first try; its output depends somewhat on the whitespace in your input. For instance, suppose you start with this:

```scala
self.test(value) && other.test(value) // the order and short-circuit are by-spec
```

scalafmt may reformat to something ugly:

```scala
self.test(value) && other.test(
    value) // the order and short-circuit are by-spec
```

But if you're dissatisfied with this, you can make scalafmt break differently by manually editing to the following:


```scala
self.test(value) && 
other.test(
    value) // the order and short-circuit are by-spec
```

scalafmt will then reformat to something more correct:

```scala
self.test(value) && 
  other.test(value) // the order and short-circuit are by-spec
```

With format-on-save enabled in your IDE, this kind of thing becomes pretty seamless:

![Peek 2020-11-03 18-23](https://user-images.githubusercontent.com/990794/98019166-b7a86880-1e01-11eb-9cfe-cc054426aea5.gif)


Once we've settled on a stable list of config rules, I can manually go over the ugliest places and fix them.